lib/morpheme.js

```diff
@@ -125,17 +125,17 @@ export default class Morpheme {
    */
   matchingValues (feature) {
     let matches = []
+
     if (feature && this.features.hasOwnProperty(feature.type)) {
       const morphemeValue = this.features[feature.type]
-      for (const featureValue of feature.values) {
-        if (morphemeValue.values.includes(featureValue)) {
-          matches.push(featureValue)
-        }
+
+      if (morphemeValue.value === feature.value) {
```

Comment:
should we put the prior logic back here? This logic is different, it will require multivalue features to match on all values, the previous version required only match on one of the values

Reply:
@balmas If we use theprevious logic - then combined gender values render not properly.
for example
![image](https://user-images.githubusercontent.com/12377640/41976413-6ffeb012-7a60-11e8-83c8-9de69eab5a92.png)
this numeral will be duplicated to 3 columns and won't render to "maculine femine neuter".
That's why I have changed the logic.

Reply:
ok. we need to test that this doesn't break the other tables though. I'm a little worried that there might have been dependencies on the prior logic that we need to make explicit.

Reply:
It also fixed rendering for pronoun
![image](https://user-images.githubusercontent.com/12377640/41976766-2e26ac48-7a61-11e8-8c32-60a6021f3075.png)

Reply:
@balmas I have checked various parts of speach and languages and didn't find errors.
But may be I have missed something.

Reply:
may be I will combine it with the prior logic - first check value = value and if it fails check through the for statement

what do you think?

Reply:
Yes that sounds like it might work. I'm not sure it's necessary. We definitely need unit tests for all of this...

Reply:
I have added both variants - first it checks value, and then if fails and the feature is Multiple, it checks in for statement.
Yes, I think that unit tests here will be really helpful and not so easy task :)